### PR TITLE
Codefix

### PR DESCRIPTION
**Pull request by ankitpws**
codefix branch merge

### CODE REVIEW

File: src/controllers/v1/form.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = class Form {
 			const createdForm = await formsHelper.create(params)
 			return createdForm
 		} catch (error) {
+			console.log(error, '</////////////')
```
Comment (rakeshSgr) [inline src/controllers/v1/form.js:26]: @nevil-mathew  remove console.log

File: src/db/questionsSet/model.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
  */
 
 const mongoose = require('mongoose')
-const { feedback } = require('@services/helper/sessions')
```
Comment (rakeshSgr) [inline src/db/questionsSet/model.js:9]: @nevil-mathew  why this change

File: src/db/sessionAttendees/queries.js [hunk 3]
```diff
@@ -66,72 +60,67 @@ module.exports = class SessionsAttendees {
...
 			}
-		})
+		} catch (error) {
+			return error
+		}
 	}
 
-	static findAllUpcomingMenteesSession(page, limit, search, filters) {
+	static async findAllUpcomingMenteesSession(page, limit, search, filters) {
 		filters.userId = ObjectId(filters.userId)
```
Comment (rakeshSgr) [inline src/db/sessionAttendees/queries.js:122]: @nevil-mathew move this line to try-catch block

File: src/generics/requests.js [hunk 4]
```diff
@@ -2,9 +2,24 @@ const request = require('request')
 const parser = require('xml2json')
 
 var get = function (url, token = '', internal_access_token = false) {
-	return new Promise(async (resolve, reject) => {
+	return new Promise((resolve, reject) => {
```
Comment (rakeshSgr) [inline src/generics/requests.js:5]: @nevil-mathew  change this to async

File: src/setupFileAfterEnv.js [hunk 5]
```diff
@@ -31,7 +31,7 @@ beforeAll(async () => {})
 
 afterAll(async () => {
 	try {
-		await db.dropDatabase()
+		// await db.dropDatabase()
```
Comment (rakeshSgr) [inline src/setupFileAfterEnv.js:34]: @nevil-mathew @ankitpws why this change ?

File: src/validators/v1/users.js [hunk 6]
```diff
@@ -7,9 +7,7 @@
 
 module.exports = {
 	pendingFeedbacks: (req) => {
-		// req.checkParams('id')
```
Comment (rakeshSgr) [inline src/validators/v1/users.js:10]: @nevil-mathew remove this if not require

File: src/db/questionsSet/model.js (hunk 2 above)
Comment (nevil-mathew) [reply src/db/questionsSet/model.js:9]: @rakeshSgr it was not used

File: src/controllers/v1/form.js (hunk 1 above)
Comment (nevil-mathew) [reply src/controllers/v1/form.js:26]: removed

File: src/generics/requests.js (hunk 4 above)
Comment (nevil-mathew) [reply src/generics/requests.js:5]: we are not using any await inside that function.so async is not required

Comment (rakeshSgr) [reply src/generics/requests.js:5]: oh ok fine

File: src/services/helper/sessions.js [hunk 7]
```diff
@@ -12,6 +12,7 @@ const kafkaCommunication = require('@generics/kafka-communication')
 const apiBaseUrl = process.env.USER_SERIVCE_HOST + process.env.USER_SERIVCE_BASE_URL
 const request = require('request')
 
+const axios = require('axios')
```
Comment (rakeshSgr) [inline src/services/helper/sessions.js:15]: @nevil-mathew why this module, are we using this ?

File: src/services/helper/sessions.js [hunk 8]
```diff
@@ -367,6 +370,7 @@ module.exports = class SessionsHelper {
 
 	static async list(loggedInUserId, page, limit, search, status) {
 		try {
+			console.log(loggedInUserId)
```
Comment (rakeshSgr) [inline src/services/helper/sessions.js:373]: @nevil-mathew  please remove the console.log

File: src/services/helper/sessions.js [hunk 9]
```diff
@@ -448,6 +452,7 @@ module.exports = class SessionsHelper {
 
 		try {
 			const session = await sessionData.findSessionById(sessionId)
+			console.log('==============>>>>>>', session)
```
Comment (rakeshSgr) [inline src/services/helper/sessions.js:455]: @nevil-mathew  please remove the console.log

File: src/services/helper/sessions.js (hunk 7 above)
Comment (nevil-mathew) [reply src/services/helper/sessions.js:15]: no, couldn't find it.
